src/Form/FieldSelect.js

```diff
@@ -73,6 +73,7 @@ class FieldSelect extends React.Component {
       <Dropdown {...classes}
         initialID={startValue.id}
         items={dropdownItems}
+        matchButtonWidth={matchButtonWidth}
```

Comment:
Shouldn't we be more dynamic in the sense where we `omit` props that we will use here and pass down the rest of the props dynamically like so `{...restOfProps}`?